src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -477,6 +477,8 @@ protected function castAttribute($key, $value)
             case 'array':
             case 'json':
                 return $this->fromJson($value);
+            case 'unserialize':
```

Comment:
I actually called it that first. Then I thought it was confusing if it would be cast TO a serialized string, so unserialize felt a bit clearer.
Should I change it?
Is the PR even something that would be accepted?
If so - just lemme know if the cast key should change and I'll fix it.
Thanks for the feedback